### ISSUE TITLE
linter-jscs freezes find & replace when replacing in files which are open

### ISSUE THREAD

**Issue by adamreisnz** [AtomLinter/linter-jscs#271]
### Version

linter-jscs: 4.0.3
Atom: 1.10.0-dev-e5a3c9e
### Issue

See https://github.com/atom/find-and-replace/issues/735

This package causes find & replace in files to hang when trying to replace text and when one of the affected files is open in a tab. It results in not all files being replaced which is a big pain.

I suspect the problem is due to linter somehow preventing the open file from being saved by the find & replace package, which causes find & replace to hang:

![image](https://cloud.githubusercontent.com/assets/490562/15624167/fd7c88bc-24d4-11e6-9110-0cbdba43c736.png)
### To reproduce

```
mkdir test
cd test
echo "test" >> test1.js
echo "test" >> test2.js
echo "test" >> test3.js
echo "test" >> test4.js
atom .
```
1. Open two of the test files in a tab
2. Open find & replace
3. Replace `test` with `abc`
4. Press find, and then replace all
5. Find & replace hangs.

**Comment by r-murphy** [comment]
I just fixed a similar issue at tidy-markdown plugin. Note that `atom.workspace.getActiveTextEditor()` returns `undefined` when the save is triggered by find & replace, since the "Project Find Results" window is opened while doing a project wide find & replace.

**Comment by Arcanemagus** [comment]
Do you have "Fix on Save" enabled?

**Comment by adambuczynski** [comment]
Yes I do

**Comment by Arcanemagus** [comment]
Marking this as a bug and help wanted, but as all JSCS development has [stopped](http://eslint.org/blog/2016/07/jscs-end-of-life) I'm not very inclined to work on this since people should be moving over to ESLint anyway.

**Comment by adambuczynski** [comment]
@Arcanemagus I have moved over to ESLint indeed :)

**Comment by Arcanemagus** [comment]
Marking this as closed, JSCS development is dead, and it seems none of the people still using it are interested in implementing this functionality.
